kcidb/db/__init__.py

```diff
@@ -270,8 +270,11 @@ def argparse_add_args(parser, database=None):
     assert database is None or isinstance(database, str)
     parser.add_argument(
         '-d', '--database',
-        help=('Database specification' if database is None
-              else f"Database specification. Default is {database!r}."),
+        help=("""DATABASE specification, formatted as <DRIVER>:<PARAMS>.
+        Driver options are bigquery:<PROJECT_ID>.<DATASET>,
+        sqlite:<DATABASE>, json:[FILE] and null.""" +
+              "" if database is None
+              else f"Default is {database!r}."),
```

Comment:
Do you think you could generate the message from the actual list of database drivers in `DRIVER_TYPES` instead?

Reply:
Absolutely!